lib/matplotlib/axes/_axes.py

```diff
@@ -201,6 +202,10 @@ def set_xlabel(self, xlabel, fontdict=None, labelpad=None, **kwargs):
             Spacing in points from the axes bounding box including ticks
             and tick labels.
 
+        loc : {'left', 'center', 'right'}, default: :rc:`xaxis.titlelocation`
```

Comment:
You need to change all this to `labellocations`  Pleas have a test that tests this, to prove that you caught all instances.